http-servers/echo-server.js

```diff
@@ -0,0 +1,9 @@
+const http = require('http');
+
+const server = http.createServer();
+
+server.on('request', (req, res) => {
+    req.pipe(res);
+});
+
+server.listen(8000);
```

Comment:
For production projects - hosts, ports etc are moved to constants or configs for different environments. The simplest - const PORT = process.env.PORT || 3000;